DataFormats/Math/interface/GeantUnits.h

```diff
@@ -27,36 +27,36 @@ namespace geant_units {
     using angle_units::operators::convertRadToDeg;
 
     // Length
-    constexpr long double operator"" _mm(long double length) { return length * 1.; }
-    constexpr long double operator"" _cm(long double length) { return length * 10.; }
-    constexpr long double operator"" _m(long double length) { return length * 1000.; }
-    constexpr long double operator"" _cm3(long double length) { return length * 1._cm * 1._cm * 1._cm; }
-    constexpr long double operator"" _m3(long double length) { return length * 1._m * 1._m * 1._m; }
-    constexpr long double operator"" _mm(unsigned long long int length) { return length * 1; }
-    constexpr long double operator"" _cm(unsigned long long int length) { return length * 10; }
+    constexpr double operator"" _mm(long double length) { return length * 1.; }
```

Comment:
Shouldn't all `long double` usage be removed?

Reply:
operator"" support only long double as argument